sdcm/sct_provision/aws/cluster.py

```diff
@@ -297,4 +309,16 @@ def _user_data(self) -> str:
         ).to_string()
 
 
+class PlacementGroup(ClusterBase):
+
+    @property
+    def _user_data(self) -> str:
+        return ''
+
+    def provision(self):
+        if self.placement_group_name:
+            create_cluster_placement_groups_aws(
+                name=self.placement_group_name, tags=self.common_tags, region=self._region(0))
```

Comment:
for multi-region this would need to iterate over regions

Reply:
same as above, we don't need this feature for multidc